tests/lib/rules/no-unneeded-ternary.js

```diff
@@ -26,6 +26,7 @@ ruleTester.run("no-unneeded-ternary", rule, {
         "var a = x === 2 ? 'Yes' : false;",
         "var a = x === 2 ? 'true' : 'false';",
         "var a = foo ? foo : bar;",
+        "var value = 'a';var canSet = true;var result = value || (canSet ? 'unset' : 'can not set')",
```

Comment:
Yes, I thought the same after pushing. There is actually no such case. I removed it.